artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ServerSessionImpl.java

```diff
@@ -1259,6 +1259,7 @@ public RoutingStatus send(final ServerMessage message, final boolean direct, boo
       return send(getCurrentTransaction(), message, direct, noAutoCreateQueue);
    }
 
+   @Override
```

Comment:
There is a way to avoid these on compile time, using Error Prone. 

I have tried that, and I even had the changes... but we have some code generated.. we would need to ignore the generated code, or to make it also have overrides.

@scop  Can we talk through IRC tomorrow, or google chat?